controllers/providers/dns/infoblox.go

```diff
@@ -106,6 +106,7 @@ func (p *InfobloxProvider) CreateZoneDelegationForExternalDNS(gslb *k8gbv1beta1.
 			}
 
 			if !reflect.DeepEqual(findZone.DelegateTo, currentList) {
+				log.Debug().Msgf("FindZone delegated records (%v)", findZone.DelegateTo)
```

Comment:
Since this message appears only in case of discrepancies, IMO it is safe to make it on the Info level instead of Debug.
Also, `findZone` is an internal implementation, I suggest replacing it with a more user-friendly explanation.  
```suggestion
				log.Info().Msgf("Found delegated zone records (%v)", findZone.DelegateTo)
```